infrastructure/cdn-in-a-box/docker-compose.yml

```diff
@@ -58,8 +58,10 @@ services:
   # defined below
   trafficops:
     build:
-      context: .
-      dockerfile: traffic_ops/Dockerfile-go
+      context: ../..
```

Comment:
Why did the context need to be expanded?

Reply:
The `trafficops` and `trafficops-perl` containers both receive a build arg named `TRAFFIC_OPS_RPM`. IMO, both of these build args should be the same value. That means that `trafficops` needs `infrastructure/cdn-in-a-box` in its build context in order to successfully be able to run its `COPY` command.

Within the constraints discussed, in order to only specify the build arg once, it would be possible for `trafficops-go` and `trafficops-perl` to both be child images of a `centos:7` image that does nothing else but receive the `TRAFFIC_OPS_RPM` build arg and export it as an environment variable. However, in this case, the `trafficops` container would still require a build context that includes `infrastructure/cdn-in-a-box`.

Reply:
I think that the consistency is certainly a good point, and it only impacts build time by about 2 minutes give or take, but you would know better than me: does this affect the size of objects stored on the filesystem or in memory in the long-term? It doesn't end up in a cache layer, but I don't know enough about the inner workings of Docker to say definitively that it isn't stored somewhere, and I was wondering if you knew.

Reply:
We can add a `.dockerignore`, list everything, and except the files we use. That should speed it back up.

Reply:
That  sounds like a great idea

Reply:
Added in 7e96aaa. For me, `trafficops-perl` built ~3.5 minutes faster and `trafficops-go` built 7 seconds faster. Note that use of these dockerignores requires Docker Buildkit. Without it, a limitation of `docker-compose` is only one dockerignore per build context. That means we would need to use a single dockerignore for all `docker-compose` projects in the repo that use the repo's base path as a build context, which IMO is not worth maintaining.

Somewhere down the road, it would be nice to update the Makefile to copy in files that CDN-in-a-Box Dockerfiles add to somewhere within the CiaB directory itself so we can use the CiaB directory as a build context for all of the images.